src/xrGame/ui/UIMap.h

```diff
@@ -54,8 +54,8 @@ class CUICustomMap : public CUIStatic, public CUIWndCallback
     virtual void Draw();
     virtual void Update();
     virtual void SendMessage(CUIWindow* pWnd, s16 msg, void* pData);
-    virtual bool IsRectVisible(Frect r);
-    virtual bool NeedShowPointer(Frect r);
+    virtual bool IsRectVisible(const Frect& r);
```

Comment:
`r` is being changed inside the function and it is wrong to use references here.